mscore/inspector/inspector.cpp

```diff
@@ -184,9 +185,19 @@ void Inspector::update(Score* s)
                         sameTypes = false;
                         break;
                         }
+                      // an articulation and an ornament
+                  if ((ee->isArticulation() && toArticulation(ee)->isOrnament() != toArticulation(element())->isOrnament()) ||
+                      // a slur and a tie
+                      (ee->isSlurTieSegment() && toSlurTieSegment(ee)->accessibleInfo() != toSlurTieSegment(element())->accessibleInfo()) ||
+                      // a breath and a caesura (or different breaths)
+                      (ee->isBreath() && toBreath(ee)->accessibleInfo() != toBreath(element())->accessibleInfo()) ||
+                      // a staff text and a system text
+                      ((ee->isStaffText() || ee->isSystemText())
+                          && (ee->type() != element()->type()) || (ee->isSystemText() != element()->isSystemText())))
```

Comment:
This here causaes a warning in MinGW `warning: suggest parentheses around '&&' within '||' [-Wparentheses]`

Reply:
See PR #5611